src/setup.erl

```diff
@@ -49,14 +50,20 @@ is_cluster_enabled() ->
         {_,_} -> ok
     end.
 
+is_single_node_enabled(Dbs) ->
```

Comment:
It's what @janl had in the code before I touched it, so I mirrored his approach. I agree true/false is better.